### PR TITLE
28 resubmit paper

### PR DESCRIPTION
**Pull request by Robinlovelace**
- **changes points 7 and 8**
- **Update .Rhistory**
- **Delete .~lock.tools.csv#**
- **Delete .Rhistory**
- **Remove js libs**
- **Test commit**

### CODE REVIEW

File: paper.qmd [hunk 1]
```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
 
 ::: -->
 ```
+## Stakeholder engagement {#sec-stakeholder-engagement}
+
+The CRUSE Tool was developed for, and in close collaboration with, practitioners in Ireland.
+Although the overall approach of the project was quantitative, stakeholder feedback and qualitative data was deemed essential to ensure the tool was fit for purpose, mitigating against the risk that the project would be 'technocentric' and a 'problem looking for a solution'.
```
Comment (eugenividal) [inline paper.qmd:610]: Great justification for the stakeholder engagement!

File: paper.qmd [hunk 2]
```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
...
+From the outset of the project, we aimed to follow @coppola2022's advocacy for a "human-centric approach in which personalization, purpose, ethics and inclusivity are key", rather than an 'off-the-shelf' product that failed to understand, let alone meet, local needs.
+As outlined in the overview of papers in the collection Inclusive and Collaborative Transport cited above, the 'digital transformation' taking place in transport planning (and many other sectors) has the potential to increase existing inequalities [@coppola2022].
```
Comment (eugenividal) [inline paper.qmd:612]: “As described in the overview of articles in the Inclusive and Collaborative Transportation collection cited above” seems unclear and wordy, I'd rephrase it to “As outlined by Coppola and Lobo (2022).”

File: paper.qmd [hunk 3]
```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
...
+A wide range of methods and measure can be taken to ensure that digital tools are inclusive \[...\].
```
Comment (eugenividal) [inline paper.qmd:613]: I'd remove the three dots and perhaps change the term "inclusive" to "are accessible to their recipients.

File: paper.qmd [hunk 4]
```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
...
+– What will change?
+– How will planning vary depending on trip purpose?
+– What sort of scenarios interested in testing?
+– How useful is having different networks for
+planning infrastructure and/or more detail,
+e.g. there could be an option for variation in
+access for professionals and public (the latter
+remaining simple) -->
+
+Due to time and skill constraints in the team developing the CRUSE tool (we lacked researchers with a background in qualitative methods), we opted for the well-tested and cost-effective approach of holding workshops with stakeholders in case study area, an approach that is documented with reference to participatory development of digital tools for supporting energy transitions [@hewitt2020].
```
Comment (eugenividal) [inline paper.qmd:643]: I’d remove "due to time and skill constraints (...)". You already mention the limitations of the qualitative approach below. So, I’d rephrase this initial sentence as follows: ‘The stakeholder engagement was carried out through workshops with stakeholders in the case study area, an approach that is documented with reference to participatory development of digital tools for supporting energy transitions’.

File: paper.qmd [hunk 5]
```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
...
+We held a total of three workshops over the course of the project and one open-ended live demo during a 2-day event in Sligo in which we collected feedback from dozens of people.
+Other than the live demo, all of the workshops took place online.
+The first two workshops took place in May 2022, in which key staff involved in cycle network planning in Kildare and Limerick counties were presented the tool.
```
Comment (eugenividal) [inline paper.qmd:646]: I find it more logical to move the workshop 1 and 2 descriptions to the third paragraph and (perhaps) describe the ‘open-ended live demo’ in a new fourth paragraph.

Comment (eugenividal) [inline paper.qmd:646]: Rephrase this sentence to “The first two workshops took place in May 2022, during which the tool was introduced to key staff involved in cycle network planning in Kildare and Limerick counties”.

File: paper.qmd [hunk 6]
```diff
@@ -821,6 +875,11 @@ Sustainable transport policies should plan for walking, wheeling and cycling, an
 This raises the question of whether other active modes should be incorporated into the results using the OD-based approach outlined in this paper, or whether different modeling approaches are needed to properly capture the shorter distance trips typically made by walking [@cooper2018].
 Furthermore, the estimates of cycling presented in the CRUSE Tool omit multi-modal trips including public transport, and omit trip chaining, due to the need to capture a large portion of cycling potential within the resource constraints of the project.
 
+A final limitation of the approach relates to *how* it was developed.
```
Comment (eugenividal) [inline paper.qmd:878]: I like it, you recognize the participatory process could have been better and explain how. Perhaps it would be good to include some reference as to why the broader the better the participation. Although, I don't think it's essential either, since it's quite obvious.

**Review by eugenividal — COMMENTED** [review]
Hi @Robinlovelace,

This looks great! I believe these changes are more than enough to persuade reviewer 1 to approve it for publication. Please review some minor comments in the PR qmd file.

I will send also a PR with changes to your changes. Approve them only if they are useful.

Also, I realised we haven't responded to comment 10 yet. Can you handle that? I believe you’ll replace the jargon much better than me.

**Comment by Robinlovelace** [comment]
Thanks for the comments @eugenividal and great job. In answer to your question:

> Also, I realised, after reading your responses (which I also find excellent), that we haven't responded to comment 10 yet. Can you handle that? I believe you’ll replace the jargon much better than me.

Yes I plan to give that a go, journey back to Leeds from Hereford :train2:

File: paper.qmd (hunk 5 above)
Comment (Robinlovelace) [reply paper.qmd:646]: :+1:

**Comment by eugenividal** [comment]
Have a nice 🚆 trip back Leeds and let me know if you need any clarification!

**Comment by eugenividal** [comment]
Final changes pushed directly to the 28-resubmit-paper branch, @Robinlovelace. I also pushed a couple of minor changes into the cruse-responses main branch. A couple of changes I couldn't make and you may consider making are:
- remove the 'references' heading in the Title page
- remove the '*corresponding author' tag on the first page from the Blinded Manuscript
- align/make consistent the citation styles in Table1 and the reference list
- make Figure 6 smaller to avoid page number 22 overlapping the figure caption. In Figure 6, the counties' names and network types font should be bigger.
Hope it helps!

**Comment by Robinlovelace** [comment]
Many thanks Eugeni!

I've made some changes:

- [x] remove the 'references' heading in the Title page                                                                 
- [ ] remove the '*corresponding author' tag on the first page from the Blinded Manuscript                              

Not sure how to fix this in .qmd but can fix in .tex version before submission

- [ ] make consistent the citation styles in Table1 and the reference list in the Manuscripts                           

I think it's fine as is so leaving for now

- [x] make Figure 6 smaller to avoid page number 22 overlapping the figure caption in the Manuscripts.

- [ ] In Figure 6, the counties' names and network types font should be bigger.                                                           

I think it's OK
 
  Hope it helps! 

Helps loads, thank you!

**Comment by eugenividal** [comment]
Hi @Robinlovelace,

Regarding [ ] remove the '*corresponding author' tag on the first page from the Blinded Manuscript, I also don't know how to fix it in .qmd. You can fix it in .tex or .pdf versions later, although I don't think it's essential.

No problem if you don't make the rest of the changes in the unchecked boxes. These are very very minor aspects that the reviewers did not ask for.
